.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,8 +29,8 @@ If applicable, add screenshots to help explain your problem.
 **Environment (please complete the following information):**
  - OS: [e.g. Windows 10]
  - Unity Version: [e.g. 2019.1]
- - MLAPI Version: [e.g. v6.0.1]
- - MLAPI Commit: [e.g. https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/commit/c102935df1d7e0928283b48948fe96e5d96dd961]
+ - Netcode Version: [e.g. v6.0.1]
+ - Netcode Commit: [e.g. https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/commit/c102935df1d7e0928283b48948fe96e5d96dd961]
```

Comment:
we'll come back to this link (`com.unity.multiplayer.mlapi`) after we rename this repository.